src/controllers/recordController.js

```diff
@@ -0,0 +1,41 @@
+const express = require("express");
+const recordService = require("../services/recordService");
+const { throwError } = require("../utils/throwError");
+const { readRecordService, createRecordService } = recordService;
+
+const readRecord = async (req, res) => {
+  try {
+    const { id } = req.params;
+    const readRecordService = await recordService.readRecordService(Number(id));
+    if (readRecordService === undefined) {
+    return res.status(400).json("NO_USER");
+    }
+    res.status(200).json(readRecordService);
+  } catch (error) {
+    console.log(error);
+  }
```

Comment:
```suggestion
const readRecord = async (req, res) => {
  try {
    const { id } = req.params;
    const recordData = await recordService.readRecord(Number(id));
    if (recordData === undefined) {
    return res.status(400).json({message: "NO_USER"});
    }
    res.status(200).json(readRecordService);
  } catch (error) {
    console.log(error);
  }
```

- 함수 이름에는 `service`가 붙지 않아도 됩니다 :) 물론 팀내 합의가 있다면 붙여도 되지만, 일반적으로 잘 붙이지 않습니다.
- JSON응답 body는 객체 형식으로 보내야 합니다.

Reply:
반영했습니다.